config.go

```diff
@@ -0,0 +1,39 @@
+package timescaledb
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type config struct {
+	// Connection URL in the form specified in the libpq docs,
+	// see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING):
+	// postgresql://[user[:password]@][netloc][:port][,...][/dbname][?param1=value1&...]
+	URL              string
+	PushInterval     time.Duration
+	ConcurrentWrites int
```

Comment:
Not using null values means that there could not be any defaults (at least in the current way we do configs), so please use the null types as it was done in the original PR.

Reply:
Done